crypto/aes/build.info

```diff
@@ -38,6 +38,8 @@ GENERATE[aesv8-armx.S]=asm/aesv8-armx.pl $(PERLASM_SCHEME)
 INCLUDE[aesv8-armx.o]=..
 GENERATE[vpaes-armv8.S]=asm/vpaes-armv8.pl $(PERLASM_SCHEME)
 
+GENERATE[aes-armv4.S]=asm/aes-armv4.pl $(PERLASM_SCHEME)
+GENERATE[bsaes-armv7.S]=asm/bsaes-armv7.pl $(PERLASM_SCHEME)
```

Comment:
Ohhhhhh!  This said it all: `Found an implicit rule for 'crypto/aes/aes-armv4.s'`

Of course, it's the `BEGINRAW`..`ENDRAW` section that needs correcting.